checks.d/spark.py

```diff
@@ -263,6 +267,11 @@ def _get_running_apps(self, instance, tags):
             cluster_mode = SPARK_YARN_MODE
 
         if cluster_mode == SPARK_STANDALONE_MODE:
+            # check for PRE-20
+            pre20 = instance.get(SPARK_PRE_20_MODE)
+            if pre20 is not None:
+                if pre20 is True:
+                    self._spark_pre_20_mode = True
             return self._standalone_init(master_address)
```

Comment:
We can maybe pass the `pre20` argument to `_standalone_init()` as an optional argument. Prototype would change to:
```
def _standalone_init(self, spark_master_address, pre20=False):
```